src/app/view/components/main-legacy/main-legacy.component.html

```diff
@@ -1,3 +1,4 @@
+<dot-dialog></dot-dialog>
```

Comment:
I move this outside the app.component because the app.component loads in the login and we don't need this in the login, so is a better place here.